Fixed date parsing by using a regex

Falydoor opened the pull request:
In daylight saving time, the timezone offset is different (+0200) so the replace was failing.
With using a regex, the replace will always work no matter the timezone offset.

Canop commented:
You can't parse a date by removing the timezone, or the date you get is wrong as soon as the browser isn't in the same timezone than the server. We need a proper parsing here.

Falydoor commented:
I did that because before this is how the date was parsed : 
```js
function findMHSeconds() {
	var text = $('body').text();
	var lines = text.split('\n');
	for (var i = lines.length; i-- > 0;) {
		var line = lines[i];
		if (line.indexOf("Page générée") > 0) {
			var t = Chrall_tokenize(line);
			var d = new Date(atoi(t[2]), atoi(t[1]) - 1, atoi(t[0]), atoi(t[3]), atoi(t[4]), atoi(t[5]));
			return d.getTime() / 1000;
		}
	}
	return 0;
}
```

So by doing that : 
```js
// new Date(year, month, day, hours, minutes, seconds)
var d = new Date(atoi(t[2]), atoi(t[1]) - 1, atoi(t[0]), atoi(t[3]), atoi(t[4]), atoi(t[5]));
```
The timezone on the page was never used with the old date parsing.

That's why I have chosen to not include the timezone when I changed this method.
And to be honest, I don't know how the server who parse the cdm handles the "seconds" parameter in this request (does he want GMT seconds ? Or he will use his own timezone ?) : 
```js
chrall.jsonp(chrall.serveurPublic() + "json?action=accept_cdm_jsonp&author=" + player.id + "&cdm=" + encodeURIComponent(cdm) + "&seconds=" + chrall.findMHSeconds());
```
